pgpy/pgp.py

```diff
@@ -255,6 +256,13 @@ def revocation_key(self):
         return None
 
     @property
+    def revocation_reason(self):
```

Comment:
a docstring, maybe? clarify that this can (or cannot?) be used to check if a key is revoked.